views/sign_in.erb

```diff
@@ -0,0 +1,17 @@
+<h1> Sign in to Chitter </h1>
+
+<form action="/sign-in" method="post">
+  <label for="email">Email</label>
+    <div>
+    <input type="text" name="email">
+    </div>
+<br>
+  <label for="password">Password</label>
```

Comment:
Using type = password would block the user/other from seeing the info when it is being typed in